app/sidecar_query_server.go

```diff
@@ -168,17 +161,14 @@ func NewSideCarQueryServer(appCodec codec.Codec, config domain.Config, logger lo
 		quotePriceUpdateWorker.RegisterListener(poolLiquidityComputeWorker)
 
 		// Initialize ingest handler and usecase
-		ingestUseCase, err := ingestusecase.NewIngestUsecase(poolsUseCase, routerUsecase, pricingSimpleRouterUsecase, tokensUseCase, chainInfoUseCase, appCodec, quotePriceUpdateWorker, logger)
+		ingestUseCase, err := ingestusecase.NewIngestUsecase(poolsUseCase, routerUsecase, pricingSimpleRouterUsecase, tokensUseCase, chainInfoUseCase, appCodec, quotePriceUpdateWorker, candidateRouteSearchDataWorker, logger)
 		if err != nil {
 			return nil, err
 		}
 
 		// Register chain info use case as a listener to the pool liquidity compute worker (healthcheck).
 		poolLiquidityComputeWorker.RegisterListener(chainInfoUseCase)
 
-		// Register chain info use case as a listener to the candidate route search data worker (healthcheck).
-		candidateRouteSearchDataWorker.RegisterListener(chainInfoUseCase)
```

Comment:
Note: the pipeline is now changed because we need candidate route data for computing prices